scripts/release/packager/PACKAGE.md

```diff
@@ -21,20 +25,33 @@ Descriptions of the files in this directory is as follows:
   XCFramework for `MapboxMaps`.
 - NOTE: This script **MUST** be called from this directory
 
-### `download-dependency-xcframeworks.sh`
-
+### `download-dependency.sh`
+- Usage:
+  ```
+  sh download-dependency.sh <SDK_REGISTRY_NAME> <SDK_REGISTRY_ARTIFACT> <VERSION>
+  ```
 - This script downloads `MapboxCoreMaps.xcframework`, `MapboxCommon.xcframework`
   and `MapboxMobileEvents.xcframework` from SDK Registry
 - NOTE: You **MUST** have a valid `.netrc` token
 
-### `create-turf-xcframework.sh`
+### `build-dependency.sh`
 
-- This script clones `turf-swift`, checks out the git tag represented in
-  `versions.json` and then creates `Turf.xcframework`.
+- Usage:
+  ```
+  sh build-dependency.sh <NAME> <GIT_REPO_URL> <GIT_TAG> <SCHEME>
+  ```
+- This script clones a given repository, checks out a `git tag`, and builds the specified `scheme`.
+- Uses `xcodebuild`
+- The `<NAME>` provided must match the name of the `.xcodeproj`.
```

Comment:
Must also match the base name of the resulting .framework product built by $SCHEME